### DISCUSSION TITLE
View navigation items based on Laravel policies

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2212] [Ideas]
Hi,

it would be great to link the users' ability to see menu groups/ items based on defined policies in Laravel. So. e.g if the user can see UserList depends on his/her ability to have a true on view_any in the UserPolicy.

See for reference: https://nova.laravel.com/releases/4.2.5.

**Comment by danharrin** [comment]
We already do that :)

**Reply by vv-ac** [reply]
Thanks, that is great. I was simply telling from the documentation and likely did overlook that.
